plugin.audio.radio.roks/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radio.roks" name="Radio Roks online" version="0.1.0" provider-name="Eugene Kuzin">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="radioroks.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <license>GNU General Public License, v3</license>
+        <summary lang="en">Online audio streams by Radio ROKS Ukraine (http://radioroks.ua)</summary>
+        <summary lang="ru">Онлайн-радиоканалы Радио РОКС Украина (http://radioroks.ua)</summary>
+        <summary lang="uk">Онлайн-радіоканали Радіо РОКС Україна (http://radioroks.ua)</summary>
+        <description lang="en">This is the unofficial Kodi add-on which adds online radio channels by Radio ROKS Ukraine to your player.</description>
+        <description lang="ru">Неофициальный add-on, добавляющий онлайн-каналы Радио РОКС Украина в ваш плеер.</description>
+        <description lang="uk">Неофиційний add-on, що добавляє онлайн-канали Радіо РОКС Україна у ваш плеєр.</description>
+        <disclaimer lang="en">The author of this add-on is not affiliated with Radio ROKS Ukraine and not responsible for any audio and video content provided.</disclaimer>
+        <disclaimer lang="ru">Автор данного дополнения никак не связан с Радио РОКС Украина и не несет ответственности за предоставляемый контент.</disclaimer>
+        <disclaimer lang="uk">Автор цього доповнення ніяк не пов'язаний з Радіо РОКС Україна та не несе відповідальності за контент, що надається.</disclaimer>
+        <email>kuzin2006@gmail.com</email>
+        <website>http://kodi-roks.kuzin.kiev.ua/</website>
+        <platform>all</platform>
+        <language>en ru uk</language>
```

Comment:
language refers to content, so probably the radio is broadcasted in ukranian language only?